javascript/javascript1/week2/student manager.js

```diff
@@ -0,0 +1,53 @@
+// Student manager
+const class07Students = [];
+function addStudentToClass(studentName) {
+    if(studentName !== "") {
+        if(class07Students.length === 0) {
+            class07Students.push(studentName);
+        } else if(class07Students.length < 6) {
+            for(let i = 0; i <= class07Students.length; i++) {
```

Comment:
nice, you can also use include(). read more about it here:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes

Reply:
Got it. Thanks :)